### PR TITLE
pathping: add page

### PR DESCRIPTION
**Pull request by pxgamer**
Part of #1579

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/windows/pathping.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# pathping
+
+> A trace route tool combining features of Ping and Tracert.
```
Comment (agnivade) [inline pages/windows/pathping.md]: We can make ping and tracert to be lowercase. And wrap them in backticks.

File: pages/windows/pathping.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+- Ping and trace the route to a host:
+
+`pathping {{hostname}}`
+
+- Do not resolve addresses to host names:
```
Comment (agnivade) [inline pages/windows/pathping.md]: hostnames to addresses ?

Comment (pxgamer) [reply pages/windows/pathping.md]: It's a bit odd, but it is addresses to hostnames.
Screenshot

Comment (pxgamer) [reply pages/windows/pathping.md]: >Or maybe `-n` prevents the reverse lookup of the hostname for the corresponding IP?

That is correct. It prevents looking up the hostname from the IP. I'm probably just getting confused.

Comment (agnivade) [reply pages/windows/pathping.md]: Ok, then perhaps - `Do not perform reverse lookup of ip-address to hostname:` is more clear.

Comment (pxgamer) [reply pages/windows/pathping.md]: Sounds good to me. :+1:

**Pull request merged**